src/main/java/org/betonquest/betonquest/quest/event/log/LogEventFactory.java

```diff
@@ -45,8 +45,8 @@ public LogEventFactory(final BetonQuestLoggerFactory loggerFactory) {
 
     @Override
     public Event parseEvent(final Instruction instruction) throws InstructionParseException {
-        final LogEventLevel level = instruction.getEnum("level", LogEventLevel.class, LogEventLevel.INFO);
-        final String raw = instruction.getInstruction();
+        final LogEventLevel level = instruction.getEnum(instruction.getOptional("level"), LogEventLevel.class, LogEventLevel.INFO);
+        final String raw = instruction.getInstruction().substring("log".length() + 2);
```

Comment:
Why +2 i think it need to be +1? can you explain, as i am sure i missed something

Reply:
Of course you are right and I messed this up. Wonder what went wrong in testing that I did not notice this.